src/style.css

```diff
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  display: block;
+  margin: 8px;
+}
+
+.form {
+  display: flex;
+  justify-content: left;
+  margin: 20px auto 0;
+  width: 100%;
+}
+
+.bigform {
+  height: 70px;
+}
+
+.smallform {
+  width: 100%;
+  height: 42px;
+}
+
+.barcontainer {
+  width: 100%;
+  position: relative;
+  display: block;
+}
+
+.barcontainer--1::after {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  background-image: url(./images/Vector.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.barcontainer--2::after {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  background-image: url(./images/Icon.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.barcontainer::after {
+  content: "";
+  position: absolute;
+  display: block;
+}
+
+.bar {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.bar:focus {
+  border-radius: 3px;
+  font-weight: 300;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.bar:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.bigbar {
+  font-family: "Avenir", sans-serif;
```

Comment:
why did you set it only for the `bigbar`? you have ` font-family: inherit;` in the `.bar` so you can delete it here

Reply:
fixed